src/deriver/consumer.py

```
@@ -45,36 +44,41 @@
+        try:
+            validated = RepresentationPayload(**payload)
+        except ValidationError as e:
+            logger.error(
+                "Invalid representation payload received: %s. Payload: %s",
+                str(e),
+                payload,
+            )
+            raise ValueError(f"Invalid payload structure: {str(e)}") from e
+        await deriver.process_representation_task(validated)
+    else:
+        raise ValueError(f"Invalid task type: {task_type}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)